### PR TITLE
Luxembourg

### PR DESCRIPTION
**Pull request by njs59**
## Summary

Run and added outputs from Luxembourg simulations

## Further work

Change plot sizes to a consistent size, this will be done

## Checklist

- [ ] All new functions have docstrings in the correct style
- [ ] I've verified the complete docs build locally without errors
- [ ] I've maintained 100% coverage (please mention any 'no cover' annotations explicitly)
- [ ] I've unit-tested all new methods directly

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] I have updated the wiki with new parameters/model functionality

## Closing issues

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -176,3 +176,6 @@ python_examples/NZ_example/NZ_inputs/*.csv
 kernprof.py
 *.prof
 *.lprof
+
+# Large csv output files
+large_csv/
```
Comment (KCGallagher) [inline .gitignore:181]: What file is this? I don't think anything should be outputted on the top level anyway - I think we should remove this, and if you want to keep this file locally, then change the path&name to something more descriptive

File: python_examples/luxembourg_example/luxembourg_inputs/luxembourg_initial_infect.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import pandas as pd
```
Comment (KCGallagher) [inline python_examples/luxembourg_example/luxembourg_inputs/luxembourg_initial_infect.py:1]: This file is missing a header to describe what it does - see older files for examples

File: python_examples/luxembourg_example/luxembourg_repeat_sims_loc_change.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
+#
```
Comment (KCGallagher) [inline python_examples/luxembourg_example/luxembourg_repeat_sims_loc_change.py:1]: This header is the same as `python_examples/luxembourg_example/luxembourg_intervention_flow.py` - if you need this as a separate file to that one, you should explain why it is different

File: python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
+#
```
Comment (KCGallagher) [inline python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst.py:1]: This file looks nearly identical to `python_examples/luxembourg_example/luxembourg_repeat_sims_loc_change.py` - please combine them and just describe the necessary modifications in the comments to run one or the other

File: python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst_interventions.py [hunk 5]
```diff
@@ -0,0 +1,73 @@
+#
```
Comment (KCGallagher) [inline python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst_interventions.py:1]: Is this the same as `python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst.py` but with the intervention parameters json? Again, just have one script, and in the comments explain how you have change json input to get both functionalities. This should also be documented in the readme

**Review by KCGallagher — CHANGES_REQUESTED** [review]
Looks good but there's still a lot of duplication, and its not clear why - we could do with a readme explaining the role of each script akin to the interventions [readme](https://github.com/SABS-R3-Epidemiology/epiabm/blob/main/python_examples/intervention_example/README.md)

What is the role of all the `day_40.png` plots - are they in the paper? (e.g. `python_examples/luxembourg_example/paper_plots/post_day_40.png`)


Large csv files that should be in the gitignore:
* `python_examples/luxembourg_example/luxembourg_inputs/luxembourg_adapted_5_in_cell_input.csv`
* `python_examples/luxembourg_example/luxembourg_inputs/luxembourg_input_file.csv`

File: .gitignore (hunk 1 above)
Comment (njs59) [reply .gitignore:181]: This is the folder holding all the 5.9 million lined csv outputs. Which all are too large to push

File: python_examples/luxembourg_example/luxembourg_inputs/luxembourg_initial_infect.py (hunk 2 above)
Comment (njs59) [reply python_examples/luxembourg_example/luxembourg_inputs/luxembourg_initial_infect.py:4]: Changed

File: python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst_interventions.py (hunk 5 above)
Comment (njs59) [reply python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst_interventions.py]: Changed

File: python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst.py (hunk 4 above)
Comment (njs59) [reply python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst.py]: Changed

File: python_examples/luxembourg_example/luxembourg_repeat_sims_loc_change.py (hunk 3 above)
Comment (njs59) [reply python_examples/luxembourg_example/luxembourg_repeat_sims_loc_change.py]: Collated into 1 file

**Review by KCGallagher — APPROVED** [review]

**Pull request merged**